tapdb/asset_minting_test.go

```diff
@@ -790,6 +872,7 @@ type randAssetCtx struct {
 	tapSiblingBytes []byte
 	tapSiblingHash  chainhash.Hash
 	mintingBatch    *tapgarden.MintingBatch
+	randomGenesis   *asset.Genesis
```

Comment:
nit: since this is the group's genesis, should we call this `groupGenesis` instead?